bangazonapi/views/profile.py

```diff
@@ -178,15 +178,15 @@ def cart(self, request):
             try:
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
-                line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
+                lineitems = OrderProduct.objects.filter(order=open_order)
```

Comment:
Take another look at this one. Instead of just renaming the variable here, you'll need to locate and remove any redundant, unnecessary lines of code. By changing the variable name, we may have covered up the issue, but we haven't really solved. Now, instead of getting duplicates, our `lineitems` are just being overwritten, which means there's still code to be removed.

Consider the output... 

Where are the url, created_date, customer, paymenttype, etc. being serialized? HINT: In one of the serializers. 

Where do the `lineitems` get serialized? HINT: In _more_ than one of the serializers.

While serializing the `lineitems`, are there any tasks being repeated unnecessarily? HINT: Yep!